pkg/cloudprovider/vm.go

```diff
@@ -99,6 +99,10 @@ func (c *VMClient) Get(rgName string, nodeName string) (compute.VirtualMachine,
 
 // UpdateIdentities updates the user assigned identities for the provided node
 func (c *VMClient) UpdateIdentities(rg, nodeName string, vm compute.VirtualMachine) error {
+	if *vm.ProvisioningState == string(compute.ProvisioningState1Deleting) {
```

Comment:
Why is this `ProvisioningState1Deleting` instead of `ProvisioningStateDeleting`?

Reply:
fixed